bin/main.ml

```diff
@@ -749,6 +788,56 @@ let resolve_targets_exn ~log common setup user_targets =
     | Ok targets ->
       targets)
 
+let fswatch_command root_path =
+  let excludes = [ {|\.#|}
+                 ; {|_build|}
+                 ; {|\.hg|}
+                 ; {|\.git|}
+                 ; {|~$|}
+                 ; {|/#[^#]*#$|}
+                 ; {|\.install$|}
+                 ]
+  in
+  let path = Path.to_string_maybe_quoted root_path in
+  let uname =
+    match Bin.which "uname" with
```

Comment:
Rather than doing this in two steps (and relying on `uname`), can we first check if `inotifywait` exists and use it if it does?

Reply:
Good point.